spk/mitmproxy/src/installer.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+# Package
+PACKAGE="mitmproxy"
+DNAME="mitmproxy"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="mitmproxy"
+GROUP="users"
```

Comment:
On DSM6, I can confirm `nobody` group is available. On my system, `domoticz` account belongs to that group. I also can mention `subliminal` is in `users` group.